std_msgs/Header header

# completion time
float32 tf

# ordered goals for robots to reach
geometry_msgs/Point[] goals